### ISSUE TITLE
Training Time

### ISSUE THREAD

**Issue by star9988rr** [pengsongyou/openscene#18]
Thanks for your wonderful work!
I use a 40G A100 to train ScanNet with a batch size of 8, but the training time is about 2 days. Is this right?
Besides, when I increase the batch size to 16 (the learning rate is also doubled), I only get 41.9 (46.0 in paper) mIoU on ScanNet. Have you tried with the batch size of 16?

**Comment by pengsongyou** [comment]
Hi @star9988rr,
> Thanks for your wonderful work! I use a 40G A100 to train ScanNet with a batch size of 8, but the training time is about 2 days. Is this right? 

I believe that is correct if you try to train for 100 epochs. However, we noticed that after 50-60 epochs one can already obtain the best performance.

> Besides, when I increase the batch size to 16 (the learning rate is also doubled), I only get 41.9 (46.0 in paper) mIoU on ScanNet. Have you tried with the batch size of 16?

I have not tried that, but probably you should not double your learning rate. You can try again with a batch size of 16 with the same learning rate.
